hack/check/.yamllintconfig.yaml

```diff
@@ -0,0 +1,12 @@
+---
+extends: relaxed
+
+rules:
+  line-length: disable
+  trailing-spaces: disable
+  new-line-at-end-of-file: disable
```

Comment:
lets begin by enabling this check and fix the related yaml files ?

Reply:
Sure I was waiting for any suggestions in the script.

Reply:
Done I have enabled trailing-spaces and new-line-at-end-of-file and updated the files accordingly.